debian/chromium-browser-appmode.install

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/dh-exec
+debian/chromium-browser-appmode.desktop usr/share/applications
+debian/chromium-browser-appmode.py => usr/bin/chromium-browser-appmode
```

Comment:
Is the `=>` syntax correct?  How is that different from just separating source and dest with a space?